src/pocketmine/plugin/PluginBase.php

```diff
@@ -304,6 +307,10 @@ public function getPluginLoader(){
 		return $this->loader;
 	}
 
+	public function getCommandSender() : PluginCommandSender{
```

Comment:
It is to allow manipulation of the permissions from other plugins. I don't know why this could be helpful, but preserving this capability is better than not doing so.